lib/components/term.js

```diff
@@ -299,11 +299,11 @@ export default class Term extends Component {
             height: '100%'
           }}
           /> :
-        <div
-          className={css('scrollbarShim')}
-          onMouseEnter={this.handleScrollEnter}
-          onMouseLeave={this.handleScrollLeave}
-          />
+            <div
```

Comment:
Hmm, maybe related to https://github.com/yannickcr/eslint-plugin-react/issues/901

That being said, the code there is not very readable. I would personally have put them in two separate variables and used the variables in the ternary instead.